dotNetInstallerLib/Component.cpp

```diff
@@ -104,6 +105,7 @@ void Component::Load(TiXmlElement * node)
 	// progress options
 	show_progress_dialog = XmlAttribute(node->Attribute("show_progress_dialog")).GetBoolValue(true);
 	show_cab_dialog = XmlAttribute(node->Attribute("show_cab_dialog")).GetBoolValue(true);
+    hide_component_if_installed = XmlAttribute(node->Attribute("hide_component_if_installed")).GetBoolValue(false);

```

Comment:
I think we should convert all tabs to spaces in all files so that it's consistent because right now it isn't consistent. I can do this in a separate pull request after this one. Is that ok?